deep-learning/_posts/2017-09-17-3dbiomed-segmen.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: review
+title:  "Joint Sequence Learning and Cross-Modality Convolution for 3D Biomedical Segmentation"
+tags:   3D Biomedical Segmentation
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/pdf/1704.07754.pdf   
+cite:
+  authors: "Kuan-Lun Tseng, Yen-Liang Lin, Winston Hsu, Chung-Yang Huang1"
+  title:   "Joint Sequence Learning and Cross-Modality Convolution for 3D Biomedical Segmentation"
+  venue:   "CVPR 2017"
+---
+
+### Description
+This paper proposes an end-to-end deep encoder-decoder network for 3D biomedical segmentation. The network is a combination of three main parts: multi-modal encoder, cross-modality convolution, and convolutional LSTM. 
```

Comment:
put 
**multi-modal encoder**, **cross-modality convolution**, and **convolutional LSTM**.